keyhandler/mykeyMessage.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
you can remove this, it's not valid.

it should be just, but I don't think it's required in the first instance, and should be unsupported.

```yaml
        code : KEY_SPACE
        type : EV_KEY
        value : 1
```

Reply:
removed